Create `run_tests.py` (#8720)
yt-dlp/yt-dlp: CONTRIBUTING.md

    python -m yt_dlp

To run all the available core tests use

>     python devscript/run_tests.py

Reviewer: ```suggestion
    python devscripts/run_tests.py
```

            'url': 'https://yourextractor.com/watch/42',
            'md5': 'TODO: md5 sum of the first 10241 bytes of the video file (use --test)',
            'info_dict': {
                # For videos, only the 'id' and 'ext' fields are required to RUN the test:
                'id': '42',
                'ext': 'mp4',
                # Then if the test run fails, it will output the missing/incorrect fields.
                # Properties can be added as:
                # * A value, e.g.:
                #     'title': 'Video title goes here',
                # * MD5 checksum; start the string with 'md5:', e.g.:
                #     'description': 'md5:098f6bcd4621d373cade4e832627b4f6',
                # * A regular expression; start the string with 're:', e.g.:
                #     'thumbnail': r're:^https?://.*\.jpg$',

>                 # * Any Python type, e.g. int or float
>             }

Reviewer 2: ```suggestion
                # * A count of elements in a list; start the string with 'count:', e.g.:
                #     'tags': 'count:10',
                # * Any Python type, e.g. int or float
            }
```

            'url': 'https://yourextractor.com/watch/42',
            'md5': 'TODO: md5 sum of the first 10241 bytes of the video file (use --test)',
            'info_dict': {
                # For videos, only the 'id' and 'ext' fields are required to RUN the test:
                'id': '42',
                'ext': 'mp4',

>                 # Then if the test run fails, it will output the missing/incorrect fields.
>                 # Properties can be added as:
>                 # * A value, e.g.:
>                 #     'title': 'Video title goes here',
>                 # * MD5 checksum; start the string with 'md5:', e.g.:
>                 #     'description': 'md5:098f6bcd4621d373cade4e832627b4f6',
>                 # * A regular expression; start the string with 're:', e.g.:
>                 #     'thumbnail': r're:^https?://.*\.jpg$',
>                 # * A count of elements in a list; start the string with 'count:', e.g.:
>                 #     'tags': 'count:10',
>                 # * Any Python type, e.g. int or float

Reviewer: ```suggestion
                # Then if the test run fails, it will output the missing/incorrect fields.
                # Properties can be added as:
                # * A value, e.g.
                #     'title': 'Video title goes here',
                # * MD5 checksum; start the string with 'md5:', e.g.
                #     'description': 'md5:098f6bcd4621d373cade4e832627b4f6',
                # * A regular expression; start the string with 're:', e.g.
                #     'thumbnail': r're:^https?://.*\.jpg$',
                # * A count of elements in a list; start the string with 'count:', e.g.
                #     'tags': 'count:10',
                # * Any Python type, e.g.
                #     'view_count': int,
```

            }
    ```
1. Add an import in [`yt_dlp/extractor/_extractors.py`](yt_dlp/extractor/_extractors.py). Note that the class name must end with `IE`.

> 1. Run `python devscripts/run_tests.py YourExtractor` (note that `YourExtractor` doesn't end with `IE`). This *should fail* at first, but you can continually re-run it until you're done. If you decide to add more than one test, the tests will then be named `YourExtractor`, `YourExtractor_1`, `YourExtractor_2`, etc. Note that tests with `only_matching` key in test's dict are not counted in. You can also run all the tests in one go with `YourExtractor_all`

Reviewer: ```suggestion
1. Run `python devscripts/run_tests.py YourExtractor`. This *may fail* at first, but you can continually re-run it until you're done. If you decide to add more than one test, the tests will then be named `YourExtractor`, `YourExtractor_1`, `YourExtractor_2`, etc. Note that tests with `only_matching` key in test's dict are not counted in. You can also run all the tests in one go with `YourExtractor_all`
```

both are supported now
